Bottom: 1b08efc20151978a5f24c054e8ab41df83a56e3d
Top:    809aec71c8c051820776057931e66d073bec580e
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-11 16:59:04 -0500

Add script to provision staging server


---
diff --git a/provision.sh b/provision.sh
new file mode 100755
index 00000000..3762e89d
--- /dev/null
+++ b/provision.sh
@@ -0,0 +1,14 @@
+#! /bin/bash
+
+#PYTHONUNBUFFERED=1 \
+     
+ANSIBLE_ENABLE_TASK_DEBUGGER=True \
+ANSIBLE_FORCE_COLOR=true \
+ANSIBLE_HOST_KEY_CHECKING=false \
+ANSIBLE_SSH_ARGS='-o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o ControlMaster=auto -o ControlPersist=60s' \
+exec ansible-playbook \
+        --connection=ssh \
+        --timeout=30 \
+        --inventory-file=ansible-playbook/staging \
+        -vv \
+        ansible-playbook/site.yml "$@"
